src/app.tsx

```diff
@@ -1,11 +1,47 @@
 import { Component } from "react"
 import * as React from "react"
 import { View } from "react-native"
+import { UserModel } from './UsersView/usermodel';
+import { Users } from './UsersView/Users';
+
+interface ComponentState {
+    userSelected: UserModel[]
+    listUsers: UserModel[]
+}
+
+interface ComponentProps {
+    
+}
 
 export class App extends Component<any, any> {
```

Comment:
We need to move all this code into PhoneMonthView instead of app.tsx

Maybe we can move it later